src/foam/u2/view/LoginView.js

```diff
@@ -0,0 +1,309 @@
+      factory: () => {
+       return {};
+      },
+      view: { class: 'foam.u2.detail.VerticalDetailView' }
+    },
+    'param',
+    {
+      class: 'String',
+      name: 'mode_',
+      hidden: true
+    },
+    {
+      class: 'String',
+      name: 'imgPath',
+      expression: function(loginVariables) {
+        return loginVariables.imgPath || '';
+      }
+    },
+    {
+      class: 'String',
+      name: 'backLinkTxt_',
+      factory: function() {
+        let temp = this.backLink_.includes('www.') ?
+          this.backLink_.substring(this.backLink_.indexOf('www.') + 4) :
+          this.backLink_;
+        return temp.includes('://') ?
+          temp.substring(temp.indexOf('://') + 3) :
+          temp;
+      }
+    },
+    {
+      class: 'String',
+      name: 'backLink_',
+      factory: function() {
+        return this.model.backLink_ ? this.model.backLink_ : this.appConfig.url;
+      },
+      hidden: true
+    }
+  ],
+
+  messages: [
+    { name: 'GO_BACK', message: 'Go to ' },
+    { name: 'MODE1', message: 'SignUp' }
+  ],
+
+  methods: [
+    function init() {
+      if ( ! this.param ) {
+        this.param = {};
+      }
+      this.param.dao_ = ! this.param.dao_ ? this.loginVariables.dao_ : this.param.dao_;
+      if ( this.mode_ === this.MODE1 ) {
+        this.param.group_ = ! this.param.group_ ? this.loginVariables.group_ : this.param.group_;
+        this.param.countryChoices_ = this.param.countryChoices_ ? this.param.countryChoices_ : ['CA', 'US'];
+        this.model = this.SignUp.create(this.param, this);
+      } else {
+        this.model = this.SignIn.create(this.param, this);
+      }
+    },
+    function initE() {
+      this.SUPER();
+      // CREATE MODEL VIEW
+      var right = this.Element.create({}, this)
+      // Header on-top of rendering model
+        .start().show(this.topBarShow).addClass('topBar-logo-Back')
+          .start('img')
+            .attr('src', this.theme.logo)
+            .addClass('top-bar-img')
+          .end()
+      .end()
+      // Title txt and Model
+        .start().addClass('title-top').add(this.model.TITLE).end()
+        .startContext({ data: this })
+          .addClass('content-form').tag(this.MODEL).br()
+        .endContext()
+      // first footer
+        .start()
+          .start('span').addClass('bold-text-with-pad').add(this.model.FOOTER_TXT).end()
+          .start('span').addClass('link')
+            .add(this.model.FOOTER_LINK)
+            .on('click', () => {
+              this.model.footerLink(this.topBarShow_, this.param);
+            })
+          .end()
+        .end()
+      // second footer
+        .start().br()
+          .start('span').addClass('bold-text-with-pad').add(this.model.SUB_FOOTER_TXT).end()
+          .start('span').addClass('link')
+            .add(this.model.SUB_FOOTER_LINK)
+            .on('click', () => {
+              this.model.subfooterLink();
+            })
+          .end()
+        .end();
```

Comment:
This footer stuff is perhaps not named correctly. They are links that exist in signUp/SignIn they are base functions for this view. Considering all previous views shared these ... not sure I agree with switching these.

![image](https://user-images.githubusercontent.com/10802216/69061634-86177100-09e7-11ea-8fb4-a8781fc9f556.png)


![image](https://user-images.githubusercontent.com/10802216/69061745-c2e36800-09e7-11ea-95f3-2eea27788f57.png)

Reply:
I'm suggesting this idea because this sub footer can be extensively different for each application. We shouldn't be limiting it to our use case. "Not a user yet?" might be a different phrase for another application OR an application might want to add an image instead of text.

Reply:
> "Not a user yet?" might be a different phrase for another application

can be set with the model's axiom(message) name: FOOTER_TXT

this model can be overridden with extending the base model and replacing the file in the applicationController that wants to use it instead EX) this.__subContext__.register(this.SignUp, 'foam.nanos.u2.navigation.SignUp');

Reply:
> OR an application might want to add an image instead of text.

can't account for every possibility. Will come to that when we come to that. This works, and follows our current design.